backend/database/migrations/2018_02_22_023241_create_transactions_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTransactionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('transactions', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('card_id')->unsigned();
+            $table->foreign('card_id')->references('id')->on('cards');
+            $table->integer('user_id')->unsigned();
+            $table->foreign('user_id')->references('id')->on('users');
```

Comment:
Lets also keep a record of the transaction price (what price the card was sold for).

You will need to add a field here to store the price as a unsignedBigInteger (64 bit).